src/classifier.py

```diff
@@ -0,0 +1,109 @@
+from glob import glob
```

Comment:
This file looks more or less duplicated from #42 (i.e. experiments with the classifier). Let's limit this PR to just the data serialization.

Reply:
Actually, these changes are valuable: they show us proper usage of the new `fontDataset` class. Closing.